packages/components/src/summary-button-list/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { __experimentalVStack as VStack, Card } from '@wordpress/components';
+import type { SummaryButtonListProps } from './types';
+
+function SummaryButtonList( { title, children }: SummaryButtonListProps ) {
```

Comment:
Should it be `SummaryButtonsList`? I saw `buttons` instead of `button` on Figma 🤣

Reply:
The [component](https://linear.app/a8c/project/component-summarybuttonlist-42bc2fa8fe66) is called `SummaryButtonList` by the DS team. There is also [ActionList](https://linear.app/a8c/project/component-actionlist-81c8ef9a97bd/overview), etc.